### PR TITLE
Add GNU cpio

### PR DESCRIPTION
**Pull request by Uni-**
I added new package: GNU Cpio.

### CODE REVIEW

**Comment by mati865** [comment]
Check appveyor fail

**Comment by Uni-** [comment]
@mati865 fixed the build error.

File: cpio/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
+pkgname=cpio
```
Comment (Alexpux) [inline cpio/PKGBUILD]: Add header

Comment (Uni-) [reply cpio/PKGBUILD]: @Alexpux What is the header? Is it okay if I insert something like `# Maintainer: Eon Jeong`?

Comment (ErichDonGubler) [reply cpio/PKGBUILD]: You'll need at least an email for contact info, but I think that you understand what @Alexpux is talking about. :)

Comment (Adsun701) [reply cpio/PKGBUILD]: @Uni- I already made a pull request that fixes the issue as described above, but if you add yourself as the maintainer, I would prefer yours over mine.

Comment (Uni-) [reply cpio/PKGBUILD]: @ErichDonGubler @Adsun701 Thank you for all your kind comments.

File: cpio/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+pkgver=2.12
+pkgrel=1
+pkgdesc="cpio - copy files to/from archives"
+arch=('i686' 'x86_64')
+url="https://www.gnu.org/software/cpio/"
+license=('GPLv3')
+depends=('libintl')
+makedepends=('gettext-devel')
+options=('staticlibs')
+source=(http://ftp.gnu.org/pub/gnu/cpio/cpio-2.12.tar.gz)
```
Comment (Alexpux) [inline cpio/PKGBUILD]: Dont hardcode version. Use pkgver variable

**Comment by Uni-** [comment]
replaced hardcoded version by pkgver and squashed all three commits.

**Pull request merged**